app/src/ui/preferences/advanced.tsx

```diff
@@ -188,6 +188,10 @@ export class Advanced extends React.Component<
       return null
     }
 
+    const notificationSettingsURL = __DARWIN__
+      ? 'x-apple.systempreferences:com.apple.preference.notifications'
+      : 'ms-settings:notifications'
```

Comment:
@sergiou87 I'm not sure, but I think the `ms-settings` protocol is supported by Windows 10 and above, so for all versions below that, the link to 'Notifications Settings' won't work 😢

Reply:
Oh, thanks! 🙇‍♂️  Shouldn't be a huge issue since we don't have many users in older versions of Windows, but I will take a look 🤞